health/health_test.go

```diff
@@ -4,19 +4,22 @@
 package health
 
 import (
-	"errors"
 	"fmt"
 	"reflect"
 	"testing"
+	"time"
 )
 
 func TestAppendWarnableDebugFlags(t *testing.T) {
 	var tr Tracker
 
 	for i := range 10 {
-		w := NewWarnable(WithMapDebugFlag(fmt.Sprint(i)))
+		w := Register(&Warnable{
```

Comment:
```
go test -count=2 tailscale.com/health
```

I bet this panics.

You probably need an unexported unregister for tests to do in defer or t.Cleanup

Reply:
Added